packs/feats/shining-oath.json

```diff
@@ -31,7 +28,45 @@
             "remaster": false,
             "title": "Pathfinder Core Rulebook"
         },
-        "rules": [],
+        "rules": [
+            {
+                "domain": "damage",
+                "key": "RollOption",
+                "label": "Retributive Strike against an undead",
+                "option": "shining-oath",
+                "predicate": [
+                    "feature:paladin"
+                ],
+                "toggleable": true
+            },
```

Comment:
```suggestion
```
Ugh, this should just be on all retributive strikes, and that should have a toggle on it to kick on automation like this or divine smite.